monitoring/controllers/loki/promtail_release.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta2
+kind: HelmRelease
+metadata:
+  name: promtail
+spec:
+  interval: 5m
+  dependsOn:
+    - name: kube-prometheus-stack
+  chart:
+    spec:
+      version: "6.x"
+      chart: promtail
+      sourceRef:
+        kind: HelmRepository
+        name: grafana-charts
+      interval: 60m
+  # https://github.com/grafana/helm-charts/blob/main/charts/promtail/values.yaml
+  values:
+    test:
+      enabled: false
```

Comment:
I was just checking this out and wondered why you disabled tests here, if I should make the same change in #47 

I'm going to come back and review your changes because it looks like there are a few things in here I should have adopted, marking mine as a draft again - thanks for doing this a long time ago! Sorry it has taken so long to review